elementary/monitor/api/alerts/alert_filters.py

```diff
@@ -20,28 +20,32 @@ def filter_alerts(
     alerts: List[AlertType],
     alerts_filter: SelectorFilterSchema = SelectorFilterSchema(),
 ) -> List[AlertType]:
-    filtered_alerts = []
+    # If the filter is on invocation stuff, it's not relevant to alerts and we return an empty list
```

Comment:
invocation filter only applies to `edr report` - if it's accidentally used with `edr monitor` we don't want to send any alerts

Reply:
But the alerts filters are opted-in rather than opted-out so the invocations are just ignored given that we don't have `if alerts_filter.invocations is not None:`, no?